config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml

```diff
@@ -3931,6 +3931,20 @@
     </details>
   </checkerFrameworkError>
 
+  <checkerFrameworkError unstable="false">
+    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocLeadingAsteriskAlignCheck.java</fileName>
+    <specifier>dereference.of.nullable</specifier>
+    <message>dereference of possibly-null reference matcher.group(1)</message>
+    <lineContent>columnNumber = OptionalInt.of(matcher.group(1).length() + 1);</lineContent>
+  </checkerFrameworkError>
```

Comment:
The code I've suggested above will eliminate this checker error.

Reply:
https://github.com/checkstyle/checkstyle/pull/14643#discussion_r1614610586

Reply:
Done. But there is new suppression because of orElseGet:

https://github.com/checkstyle/checkstyle/blob/4c812934107632c64377b91dc685c1a082c02fce/config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml#L3941-L3946